query/src/sql/planner/format/display_rel_operator.rs

```diff
@@ -201,9 +202,9 @@ pub fn format_physical_scan(
 pub fn format_project(
     f: &mut std::fmt::Formatter<'_>,
     _metadata: &MetadataRef,
-    _op: &Project,
+    op: &Project,
 ) -> std::fmt::Result {
-    write!(f, "Project")
+    write!(f, "Project: [{}]", op.columns.iter().join(","))
```

Comment:
I added the index info because I thought the user could infer the column name based on the index, otherwise, the project would look bare. Btw, `ColumnIndex` will help developers to debug.

Reply:
I prefer to print the name of `ColumnEntry` instead.

Reply:
fixed